mango/clients/arcgis.py

```diff
@@ -184,3 +246,68 @@ def get_origin_destination_matrix(
         ]
 
         return results
+
+    def _get_origin_destination_matrix_sync(
+        self,
+        *,
+        origins: dict,
+        destinations: dict,
+        travel_mode: dict = None,
+        sleep: int = 2,
+    ):
+        """
+        Get the origin destination matrix for a list of origins and destinations in sync mode.
+        For more information about the request and the parameters, see:
+        https://developers.arcgis.com/rest/network/api-reference/origin-destination-cost-matrix-synchronous-service.htm
+
+        :param dict origins: a dict with the features and attributes of the origins
+        :param dict destinations: a dict with the features and attributes of the destinations
+        :param dict travel_mode: the travel mode configuration
+        :return: list of dictionaries with the origin destination matrix with distance in meters and time in seconds
+        :rtype: list
+        :doc-author: baobab soluciones
+        """
+        url = (
+            f"{ARCGIS_ODMATRIX_DIRECT_URL}?f=json&token={self.token}"
+            f"&origins={json.dumps(origins, separators=(',', ':'))}"
+            f"&destinations={json.dumps(destinations, separators=(',', ':'))}"
+            f"&travelMode={json.dumps(travel_mode, separators=(',', ':'))}"
+            f"&impedanceAttributeName=Minutes"
+        ).replace("%3A", ":")
+
+        url = requests.Request("GET", url).prepare().url
+
+        response = requests.get(url=url)
+        if response.status_code != 200:
+            raise Exception(f"Wrong status: {response.status_code}")
+
+        if "error" in response.json().keys():
+            raise Exception(f"Error on calculation: {response.json()['error']}")
+
+        data = response.json()["odCostMatrix"]
+        results = []
+
+        for origin in origins["features"]:
+            for destination in destinations["features"]:
+                try:
+                    results.append(
```

Comment:
it may be a little cleaner like this:
```
origin_id = str(origin["attributes"]["ObjectID"])
destination_id = str(destination["attributes"]["ObjectID"])
results.append(
    {
        "origin": origin["attributes"]["Name"],
        "destination": destination["attributes"]["Name"],
        "distance": data[origin_id][destination_id][2] * 1000,
        "time": data[origin_id][destination_id][0]* 60,
    }
)                 
````